File: Eren Yeager and Survey Corps.java
import java.util.Scanner;
import java.util.Arrays;

public class Main{
    public static void main(String args[]){
      
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int []k = new int[n];
      
        for(int i=0; i<n; i++){
            k[i] = 0;
        }
      
        for(int i=0; i<3; i++){
            for(int j=0; j<n; j++){
                k[j]+=sc.nextInt();
            }
        }
        
        int e = k[0];
        Arrays.sort(k);
        
        if(k[n-1] == e || k[n-2] == e || k[n-3] == e){
            System.out.println("YES");
        }
      
        else{
            System.out.println("NO");
        }
    }
}

File: Eren Yeager and Survey Corps.py
n=int(input())
k=[]

for _ in range(n):
    k.append(0)
    
for _ in range(3):
    a=list(map(int,input().split()))
    for i in range(n):
        k[i]+=a[i]
        
e = k[0]
k.sort()
k.reverse()

if(e in k[:3]):
    print("YES")
else:
    print("NO")
